packages/wallets/src/features/cfd/modals/JurisdictionModal/JurisdictionModal.tsx

```diff
@@ -18,25 +23,42 @@ const JurisdictionModal = () => {
 
     const capitalizedMarketType = MarketTypeToTitleMapper[marketType];
 
+    const toggleDynamicLeverage = useCallback(() => {
+        setIsDynamicLeverageVisible(!isDynamicLeverageVisible);
+    }, [isDynamicLeverageVisible, setIsDynamicLeverageVisible]);
+
+    const jurisdictionTitle = `Choose a jurisdiction for your Deriv MT5 ${capitalizedMarketType} account`;
+
+    const modalFooter = isDynamicLeverageVisible
+        ? undefined
+        : () => (
```

Comment:
```suggestion
    const modalFooter = isDynamicLeverageVisible &&
```
no need to explicitly return undefined in this case.

Reply:
renderFooter supports a function or undefined, this will result in result in modalFooter to be boolean or function.